src/ape_ethereum/transactions.py

```diff
@@ -243,13 +243,14 @@ def method_called(self) -> Optional[MethodABI]:
     @cached_property
     def source_traceback(self) -> SourceTraceback:
```

Comment:
should this instead be optional, and return None if it can't?

Reply:
I think it always can, for the most part. If you don't have tracing, you still get some top-level data.
However, it does fail now and then for some random node-related reasons on providers _with_ tracing support, hence the handling